humans/chain.json

```diff
@@ -0,0 +1,257 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "humans",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://humans.ai/",
+  "pretty_name": "humans",
+  "chain_id": "humans_1089-1",
+  "bech32_prefix": "humans",
+  "node_home": "$HOME/.humansd",
+  "daemon_name": "humansd",
+  "key_algos": ["ethsecp256k1"],
+  "extra_codecs": ["ethermint"],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "aheart",
+        "fixed_min_gas_price": 250000000,
+        "low_gas_price": 20000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 40000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "aheart"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/humansdotai/",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": ["v1.0.0"],
+    "cosmos_sdk_version": "0.46.11",
+    "consensus": {
```

Comment:
can we copy the consensus data into the v1.0.0 version object below in the array, please?

Reply:
done, thank you!